src/main/webapp/scripts/gallery.js

```diff
@@ -2,24 +2,59 @@
 // May need to use different libraries due to commercial use requirement
 // Masonry is released under the MIT license: https://desandro.mit-license.org/
 
-// Initialize Masonry
-console.log("initializing masonry");
-var $grid = $('.grid').masonry({
-    itemSelector: '.grid-item',
-    percentPosition: true,
-    columnWidth: '.grid-sizer',
-    horizontalOrder: true
+// global variables
+var filters = new Map()
+var $active_filters_ul = $('.active-filters');
+var $grid = $('#grid')
+
+// Call these functions when page loads
+$(document).ready(function() {
+    // Initialize Masonry
+    console.log("initializing masonry");
+    $grid.masonry({
+        itemSelector: '.grid-item',
+        percentPosition: true,
+        // horizontalOrder: true,
+        columnWidth: '.grid-sizer'
+    });
+
+    // get all data
+    createGallery("common_name");
+
+    // hides filters on click
+    deleteFilter()
 });
 
-fetchAllSpeciesData();
+function createGallery(sortBy) {
+    console.log("create");
+    fetchAllSpeciesData(sortBy);
+    console.log("loaded");
+    hideAllClasses();
+}
 
-// Test function that fetches sample JSON and appends each species to the gallery
-function fetchAllSpeciesData() {
-    fetch("/allData").then(response => response.json()).then(speciesData => {
+/** 
+ *Fetches sample JSON and appends each species to the gallery
+ * @param sortBy: the parameter to sort by
+ */
+function fetchAllSpeciesData(sortBy) {
+    const parameters = {'sortBy': sortBy};
+    const url = createQueryString("/allData", parameters);
+    console.log(url)
+    clearGallery();
+    
+    fetch(url).then(response => response.json()).then(speciesData => {
         for (var species in speciesData) {
             // Append images to grid
-            var $html = $('<div class="grid-item"> <img src="'+ speciesData[species].imageLink +
-                          '" /> <div class="overlay">' + speciesData[species].commonName + '</div> </div>'); 
+            var $html = $(
+                '<div class="grid-filters ' + speciesData[species].status + ' ' + speciesData[species].trend + ' ' + speciesData[species].taxonomicPath.order_t + ' ' + speciesData[species].taxonomicPath.class_t + '">' +
```

Comment:
Optional:
You might consider using template literals here to build the HTML string.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals